jsapp/js/components/formLanding.es6

```diff
@@ -423,23 +423,26 @@ export class FormLanding extends React.Component {
   }
   renderLanguages (canEdit) {
     let translations = this.state.content.translations;
-    if (!translations || translations.length < 2)
-      return false;
 
     return (
       <bem.FormView__cell m={['columns', 'padding', 'bordertop']}>
         <bem.FormView__cell m='translation-list'>
           <strong>{t('Languages:')}</strong>
           &nbsp;
-          <ul>
-            {translations.map((langString, n)=>{
-              return (
-                <li key={n}>
-                  {langString || t('Unnamed language')}
-                </li>
-              );
-            })}
-          </ul>
+          {!translations || translations.length < 2 &&
+            t('This project has no languages defined yet')
```

Comment:
Am I correct in assuming that `t(...)` is somehow translating the text?

Reply:
Yes 👍

Reply:
I'd expect to see `This project has no languages defined yet` if the asset content contains this:
```json
"translations": [
    null
],
```
...and that works. :heavy_check_mark:
**However**, I don't think that this should show the same thing:
```json
"translations": [
    "English (en)"
],
```
It's jarring to see `This project has no languages defined yet` if I've just gone into "Manage Languages" and named my language.

Reply:
@jnm good catch - fixed :)